src/ui/public/notify/notify.js

```diff
@@ -70,18 +78,53 @@ if (!!metadata.kbnIndex) {
   });
 }
 
+let bannerId;
+let bannerTimeoutId;
+
 function applyConfig(config) {
   Notifier.applyConfig({
-    bannerLifetime: config.get('notifications:lifetime:banner'),
     errorLifetime: config.get('notifications:lifetime:error'),
     warningLifetime: config.get('notifications:lifetime:warning'),
     infoLifetime: config.get('notifications:lifetime:info')
   });
 
-  const banner = config.get('notifications:banner');
+  // Show user-defined banner.
+  const bannerContent = config.get('notifications:banner');
+  const bannerLifetime = config.get('notifications:lifetime:banner');
+
+  if (typeof bannerContent === 'string' && bannerContent.trim()) {
+    const BANNER_PRIORITY = 100;
+
+    const dismissBanner = () => {
+      banners.remove(bannerId);
+      clearTimeout(bannerTimeoutId);
+    };
+
+    const banner = (
+      <EuiCallOut
+        title="Attention"
```

Comment:
Is "Attention" placeholder text?  It would be better if developers could put something specific here, as in the banner in the first screenshot that uses a specific title followed by a description.

Reply:
This makes sense, but it requires creating an advanced setting which users can customize, which I think is beyond the scope of this PR.

Reply:
@spalger Correct.